microprofile/testing/junit5/src/main/java/io/helidon/microprofile/testing/junit5/Configuration.java

```diff
@@ -33,7 +33,7 @@
      * If set to {@code true}, the existing (or default) MicroProfile configuration would be used. In this case it is
      * important to set property {@code mp.initializer.allow=true} in order CDI container to start, when used with
      * {@link HelidonTest}.
-     * By default uses a configuration constructed using all {@link io.helidon.microprofile.tests.junit5.AddConfig}
+     * By default uses a configuration constructed using all {@link AddConfig}
```

Comment:
Should we keep the full names in comments (I was addressed the same issue in my PRs)

Reply:
As long as it is from the same package, it is OK. The reason we do this is to have `import` statements on a class file only if used in code. As this does not introduce an import, I consider this OK

Reply:
Nevertheless this was the result of Idea refactoring, I did not want to do it...

Reply:
Fixing